azurerm/internal/services/digitaltwins/digital_twins_instance_resource.go

```diff
@@ -85,99 +87,94 @@ func resourceArmDigitalTwinsCreate(d *schema.ResourceData, meta interface{}) err
 
 	future, err := client.CreateOrUpdate(ctx, resourceGroup, name, properties)
 	if err != nil {
-		return fmt.Errorf("creating Digital Twins %q (Resource Group %q): %+v", name, resourceGroup, err)
+		return fmt.Errorf("creating Digital Twins Instance %q (Resource Group %q): %+v", name, resourceGroup, err)
 	}
 
 	if err := future.WaitForCompletionRef(ctx, client.Client); err != nil {
-		return fmt.Errorf("waiting on creating future for Digital Twins %q (Resource Group %q): %+v", name, resourceGroup, err)
+		return fmt.Errorf("waiting for creation of the Digital Twins Instance %q (Resource Group %q): %+v", name, resourceGroup, err)
 	}
 
-	resp, err := client.Get(ctx, resourceGroup, name)
+	_, err = client.Get(ctx, resourceGroup, name)
 	if err != nil {
-		return fmt.Errorf("retrieving Digital Twins %q (Resource Group %q): %+v", name, resourceGroup, err)
-	}
-
-	if resp.ID == nil || *resp.ID == "" {
-		return fmt.Errorf("empty or nil ID returned for Digital Twins %q (Resource Group %q) ID", name, resourceGroup)
+		return fmt.Errorf("retrieving Digital Twins Instance %q (Resource Group %q): %+v", name, resourceGroup, err)
 	}
 
-	id, err := parse.DigitalTwinsID(*resp.ID)
-	if err != nil {
-		return err
-	}
 	d.SetId(id.ID(subscriptionId))
```

Comment:
we'll be removing this `subscriptionId` field - so whilst this isn't _technically_ an issue - we may as well switch this out for an empty string early since we're making another change here (it's provided on the constructor above now)

```suggestion
	d.SetId(id.ID(""))
```